Aerolyzer/app/views.py

```diff
@@ -59,16 +62,87 @@ def gallery(request):
 
 @login_required
 def upload(request):
-    return render(request,
-    'app/upload.html',
-	{ 'user': request.user },
-    )
+	if request.method == 'POST' and request.FILES['myfile']:
+		myfile = request.FILES['myfile']
+		fs = FileSystemStorage()
+		filename = fs.save(myfile.name, myfile)
+		uploadedFileUrl = fs.url(filename)
+		request.session['filename'] = filename
+		request.session['uploadedFileUrl'] = uploadedFileUrl
+		# verified = imgRestFuncs(filename)
```

Comment:
@kwhitehall Something like this (with the filepath corresponding to production path instead of my machine)?
```
	request.session['uploadedFileUrl'] = uploadedFileUrl
	f = open("C:/Python271/Aerolyzer_App/Aerolyzer/" + uploadedFileUrl, 'rb')
	tags = exifread.process_file(f)
	verified = imgRestFuncs(tags)
```